in-depth/submission_guidelines.md

```diff
@@ -3,31 +3,31 @@ layout: docs
 title: Theme & Plugin Submission Guidelines
 headline: Theme & Plugin Submission Guidelines
 description: |
-  Submitting your Theme or Plugin for inclusion on the website is very easy.<br>
+  Submitting your theme or plugin for inclusion on the website is very easy.<br />
```

Comment:
Simply because the HTML5 standard still allows both and there are more working [XML](https://en.wikipedia.org/wiki/XML) parsers out there than [SGML](https://en.wikipedia.org/wiki/Standard_Generalized_Markup_Language) parsers (XML is a more strict subset of SGML). One of the reasons for this is because XML parsers can be generic, whereas SGML parsers must know the exact standard to use, e.g. HTML (`<br>` is ambiguous) :wink: A browser isn't the only piece of software that might want to parse a website's source code. Or to cut a long story short: Because developer things :laughing: